README.md

```diff
@@ -1,4 +1,8 @@
 # CITF
+
+[![Build Status](https://travis-ci.org/openebs/CITF.svg?branch=master)](https://travis-ci.org/openebs/CITF)
+[![Go Report](https://goreportcard.com/badge/github.com/openebs/CITF)](https://goreportcard.com/report/github.com/openebs/CITF)
+
```

Comment:
Can you add `go doc` and `license` batch as well

Reply:
Sure, we can !
I did not added license batch since we already didn't had license file !
However, I will do it now.